docs/source/libraries.rst

```diff
@@ -7,49 +7,53 @@ Here is where the Libraries API's descriptions are located:
 
-+-------------+-------------+---------+--------------------+
-| Peripheral  | Arduino API | ESP-IDF | Comment            |
-+=============+=============+=========+====================+
-| ADC         | Yes         | Yes     |                    |
-+-------------+-------------+---------+--------------------+
-| DAC         | No          | No      |                    |
-+-------------+-------------+---------+--------------------+
-| GPIO        | Yes         | Yes     |                    |
-+-------------+-------------+---------+--------------------+
-| SDIO/SPI    | Yes         | Yes     |                    |
-+-------------+-------------+---------+--------------------+
-| I2C         | Yes         | Yes     |                    |
-+-------------+-------------+---------+--------------------+
-| I2S         | No          | Yes     | In Development     |
-+-------------+-------------+---------+--------------------+
-| Wi-Fi       | Yes         | Yes     |                    |
-+-------------+-------------+---------+--------------------+
-| Bluetooth   | Yes         | Yes     |                    |
-+-------------+-------------+---------+--------------------+
-| RMT         | Yes         | Yes     |                    |
-+-------------+-------------+---------+--------------------+
-| Touch       | Yes         | Yes     | ESP32 & ESP32-S2   |
-+-------------+-------------+---------+--------------------+
-| Timer       | Yes         | Yes     |                    |
-+-------------+-------------+---------+--------------------+
-| UART        | Yes         | Yes     |                    |
-+-------------+-------------+---------+--------------------+
-| Hall Sensor | Yes         | Yes     | Only for ESP32     |
-+-------------+-------------+---------+--------------------+
-| LEDC        | Yes         | Yes     |                    |
-+-------------+-------------+---------+--------------------+
-| Motor PWM   | No          | No      |                    |
-+-------------+-------------+---------+--------------------+
-| TWAI        | No          | No      |                    |
-+-------------+-------------+---------+--------------------+
-| Ethernet    | Yes         | Yes     | Only for ESP32     |
-+-------------+-------------+---------+--------------------+
-| USB         | Yes         | Yes     | Only for ESP32-S2  |
-+-------------+-------------+---------+--------------------+
++---------------+---------------+---------------+---------------+--------------------+
+| Peripheral    | ESP32         | ESP32-S2      | ESP32-C3      | Comments           |
++===============+===============+===============+===============+====================+
+| ADC           | Yes           | Yes           | Yes           |                    |
++---------------+---------------+---------------+---------------+--------------------+
+| Bluetooth     | Yes           | Not Supported | Yes           |                    |
```

Comment:
I wonder if we should specify what type of BT is supported. C3 is BLE-Only, while the ESP32 also supports classic BT (BTSerial, A2DP, etc.)